keyboards/keebio/bdn9/keymaps/vosechu-browser/keymap.c

```diff
@@ -0,0 +1,72 @@
+#include QMK_KEYBOARD_H
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
```

Comment:
`_______` and `XXXXXXX` are in QMK core already, so you don't need to define them here.